scripts/train/train.py

```diff
@@ -262,10 +262,14 @@ def main(cfg: DictConfig) -> Trainer:
                                             'save_folder',
                                             must_exist=False,
                                             default_value=None)
-    save_latest_filename: str = pop_config(cfg,
-                                           'save_latest_filename',
-                                           must_exist=False,
-                                           default_value='latest-rank{rank}.pt')
+    is_state_dict_sharded: bool = (fsdp_config.get('state_dict_type', 'sharded')
```

Comment:
isn't the default `full`?

Reply:
no we're changing the default to sharded I think

Reply:
ah no, the intention is not to change the default in llm foundry, just in the finetuning api. sorry, was not clear about this